wire-tests/src/test/proto-kotlin/com/squareup/wire/protos/kotlin/OneOfMessage.kt

```diff
@@ -19,8 +20,29 @@ import okio.ByteString
 /**
  * It's a one of message.
  */
-data class OneOfMessage(val choice: Choice? = null, val unknownFields: ByteString =
-    ByteString.EMPTY) : Message<OneOfMessage, OneOfMessage.Builder>(ADAPTER, unknownFields) {
+data class OneOfMessage(
+  /**
+   * What foo.
+   */
+  @field:WireField(tag = 1, adapter = "com.squareup.wire.ProtoAdapter#INT32") val foo: Int? = null,
```

Comment:
I thought latest kotlin poet would pretify those for us, no?

Reply:
It hasn't been merged yet